Documentation/guideline/csharp_coding_standards.md

```diff
@@ -105,27 +105,27 @@ The class definition contains class members in the following order, from *less*
   2. `if (a == b)`
 
 ### Performace Consideration
-* **DO** use `sealed` for private classes if it is not to be inherited.
-* **DO** add `readonly` to fields if they are not tend to be changed.
-* **DO** use `static` methods if it is not instance relavant.
+* **DO** use `sealed` for private classes if they are not to be inherited.
+* **DO** add `readonly` to fields if they do not tend to be changed.
+* **DO** use `static` methods if it is not instance relevant.
 * **DO** use `RegexOptions.Compiled` for `readonly` `Regex`.
 
 ### Cross-platform coding
-Our code should supports multiple operating systems. Don't assume we only run (and develop) on Windows. Code should be sensitvie to the differences between OS's. Here are some specifics to consider.
+Our code can and should support multiple operating systems in addition to Windows. Code should be sensitvie to the differences between Operating Systems. Here are some specifics to consider:
 
 * **DO** use `Enviroment.NewLine` instead of hard-coding the line break instead of `\r\n`, as Windows uses `\r\n` and OSX/Linux uses `\n`.
 
 > **Note**
 >
-> Be aware that thes line-endings may cause problems in code when using `@""` text blocks with line breaks.
+> Be aware that these line endings may cause problems in code when using `@""` text blocks with line breaks.
 
-* **DO** Use `Path.Combine()` or `Path.DirectorySeparatorChar` to separate directories. If this is not possible (such as in scripting), use a forward slash `/`. Windows is more forgiving than Linux in this regard.
+* **DO** Use `Path.Combine()` or `Path.DirectorySeparatorChar` to separate directories. If this is not possible (such as in scripting), use a forward slash `/`. Windows is more flexible than Linux in this regard.
```

Comment:
*forgiving* has the meaning that windows can be compatible with both `/` and `\`. *forgiving* sounds better